fix: Format of key/cert upon creation of ext kafka integration endpoint (#587)
aiven/aiven-docs: docs/products/flink/howto/ext-kafka-flink-integration.md

        and ports to connect to.
    -   `security_protocol`: The type of security protocol to use
        for the connection, which is `SASL` in this case.
    -   `ssl_ca_cert`: The content of the SSL CA certificate.
    -   `ssl_client_cert`: The content of the SSL client certificate.
    -   `ssl_client_key`: The content of the SSL client key.
    -   `ssl_endpoint_identification_algorithm`: The endpoint
        identification algorithm to use for SSL verification. For
        example, `https`.

:::important

> After downloading your keys/certificates, the cypher should be in its own line
> while the PEM markers delimited by a line feed, as per [RFC 1421](https://www.rfc-editor.org/rfc/rfc1421#section-4.4).

Reviewer: ```suggestion
After downloading your keys or certificates, ensure the cipher is on its own line, 
and the PEM markers are delimited by a line feed, following the 
guidelines in [RFC 1421](https://www.rfc-editor.org/rfc/rfc1421#section-4.4).
```

Author: done

> The following bash command should help producing the expected content:

Reviewer: ```suggestion
Use the following bash command to format the content correctly:
```

Author: done
